File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/d/ems/ems/EmsCrtDTO.java
package iih.ci.ord.d.ems.ems;

import iih.bd.srv.ems.d.SrvMatchEmsRstDTO;
import iih.ci.ord.d.ems.base.BaseCiDTO;

public class EmsCrtDTO extends BaseCiDTO {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	/**
	 * 医疗单主服务id
	 * @return String
	 */
	public String getId_srv() {
		return ((String) getAttrVal("Id_srv"));
	}
	/**
	 * 医疗单主服务id
	 * @param mainSrv
	 */
	public void setId_srv(String mainSrv) {
		setAttrVal("Id_srv", mainSrv);
	}
	
	/**
	 * 医疗单物品id
	 * @return String
	 */
	public String getId_mm() {
		return ((String) getAttrVal("Id_mm"));
	}
	/**
	 * 医疗单物品id
	 * @param mainSrv
	 */
	public void setId_mm(String mainSrv) {
		setAttrVal("Id_mm", mainSrv);
	}
	
	
	/**
	 * 医疗单
	 * @return EmsMatchRstDTO
	 */
	public SrvMatchEmsRstDTO getEmsMgrInfo() {
		return ((SrvMatchEmsRstDTO) getAttrVal("EmsMgrInfo"));
	}
	/**
	 * 医疗单
	 * @param mainSrv
	 */
	public void setEmsMgrInfo(SrvMatchEmsRstDTO EmsMgrInfo) {
		setAttrVal("EmsMgrInfo", EmsMgrInfo);
	}
	
	/**
	 * 医疗单驱动
	 * @return String
	 */
	public String getEmsDriver() {
		return ((String) getAttrVal("EmsDriver"));
	}
	/**
	 * 医疗单驱动
	 * @param mainSrv
	 */
	public void setEmsDriver(String EmsDriver) {
		setAttrVal("EmsDriver", EmsDriver);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/emsmain/ems/EmsCrtDTO.cs
using iih.bd.srv.ems.d;
using iih.ci.iih.ci.ord.dto.emsmain;
using iih.ci.ord.ems.d;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using xap.mw.core.data;

namespace iih.ci.ord.dto.emsmain
{
    public class EmsCrtDTO : BaseCiDTO
    {
        /**
	 * 医疗单主服务id
	 * @return String
	 */
        public String Id_srv
        {
            get { return getAttrVal<String>("Id_srv", null); }
            set { setAttrVal<String>("Id_srv", value); }
        }


        /**
         * 医疗单物品id
         * @return String
         */
        public String Id_mm
        {
            get { return getAttrVal<String>("Id_mm", null); }
            set { setAttrVal<String>("Id_mm", value); }
        }


        /**
	 * 医疗单 --- 后续优化掉该属性
	 * @return EmsMatchRstDTO
	 */
        public SrvMatchEmsRstDTO EmsMgrInfo
        {
            get { return getAttrVal<SrvMatchEmsRstDTO>("EmsMgrInfo", null); }
            set { setAttrVal<SrvMatchEmsRstDTO>("EmsMgrInfo", value); }
        }


        /// <summary>
        /// 医疗单驱动
        /// </summary>
        public String EmsDriver
        {
            get { return getAttrVal<String>("EmsDriver", null); }
            set { setAttrVal<String>("EmsDriver", value); }
        }

    }
}
